lib/BloqadeExpr/src/Operator/Operator.jl

```diff
@@ -0,0 +1,23 @@
+module Operator
```

Comment:
Operator sounds like a confusing name with YaoBlocks's staff, let's use a more restrictive word for this? maybe just `LowLevel`?

Reply:
This is changed in the new commit. module rename Operator -> Lowlevel